### DISCUSSION TITLE
Issue encountered when Installing maven dependencies this repository as the Could not find artifact org.sunbird:analytics-core:jar:2.0

### DISCUSSION THREAD

**Discussion by muzafferjoya** [Sunbird-Obsrv/Community#152] [Issues]
`mvn clean install -DskipTests -DCLOUD_STORE_GROUP_ID=org.sunbird -DCLOUD_STORE_ARTIFACT_ID=cloud-store-sdk_2.12 -DCLOUD_STORE_VERSION=1.4.0`

[INFO] Reactor Summary for sunbird-analytics-service 2.0:
[INFO] 
[INFO] sunbird-analytics-service .......................... SUCCESS [  0.228 s]
[INFO] analytics-api-core ................................. FAILURE [ 27.895 s]
[INFO] analytics-api ...................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  28.724 s
[INFO] Finished at: 2024-04-05T10:50:16Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project analytics-api-core: Could not resolve dependencies for project org.sunbird:analytics-api-core:jar:2.0: The following artifacts could not be resolved: org.sunbird:analytics-core:jar:2.0 (absent): Could not find artifact org.sunbird:analytics-core:jar:2.0 in jcenter-repo (https://jcenter.bintray.com/) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :analytics-api-core

**Comment by sowmya-dixit** [comment]
@muzafferjoya - Looks like you have not built sunbird-analytics-core repository before building sunbird-analytics-service? If not please clone this [repository](https://github.com/Sunbird-Obsrv/sunbird-analytics-core) and build it first. 

Let us know if you still face any issues.
